dubbo-common/src/main/java/org/apache/dubbo/config/AbstractInterfaceConfig.java

```diff
@@ -221,7 +221,7 @@
     /**
      * The url of the reference service
      */
-    protected final transient List<URL> urls = new ArrayList<URL>();
+    protected final transient List<URL> urls = new ArrayList<>();
```

Comment:
**Security Issue**: The change from 'new ArrayList<URL>()' to 'new ArrayList<>()' in the declaration of 'urls' uses diamond syntax for cleaner code. This modification does not introduce a security issue directly. However, the context in which this list is used should be reviewed to ensure that URL objects added to this list are validated to prevent any potential security vulnerabilities, such as URL redirection, SSRF, or information disclosure vulnerabilities. <br> **Fix**: Ensure that any URL objects added to the 'urls' list are properly validated against a whitelist of allowed URLs or sanitized to prevent malicious inputs. <br> **Code Suggestion**: 
 ```
 -    protected final transient List<URL> urls = new ArrayList<URL>();
 +    protected final transient List<URL> urls = new ArrayList<>();

 Ensure that any URL objects added to the 'urls' list are properly validated against a whitelist of allowed URLs or sanitized to prevent malicious inputs.
 ```